js/renovation/dist/button.j.jsx

```diff
@@ -53,6 +53,10 @@ class Button extends Widget {
             }
         });
 
+        props.onContentReady = this._createActionByOption('onContentReady', {
+            excludeValidators: ['disabled', 'readOnly']
+        });
+
```

Comment:
Why we are using this code twice?

Reply:
The same was implemented for `onClick` property
In fact, I think we can get rid of these properties for widget

Reply:
Keeping this in the `Widget` maybe more convenient for us instead of in the `Button`

Reply:
If I remove this from Button - it doesn't work correctly. `onContentReady` from Widget rewrite `onContentReady` from Button's props.
If I remove this from Widget - it still work.